src/ScreenSyncOverlay.cpp

```diff
@@ -97,8 +97,15 @@ void ScreenSyncOverlay::UpdateText()
 	set_adjustments.SetParam("visible", visible);
 	if(visible)
 	{
-		float fNew = PREFSMAN->m_fGlobalOffsetSeconds;
-		float fOld = AdjustSync::s_fGlobalOffsetSecondsOriginal;
+		float fNew;
```

Comment:
Unrelated to this PR (separate fix).

Visual text displayed on ScreenSyncOverlay was just wrong for song offset